app/Phragile/ActionHandler/SprintStoreActionHandler.php

```diff
@@ -0,0 +1,130 @@
+<?php
+namespace Phragile\ActionHandler;
+
+use Phragile\PhabricatorAPI;
+use \Flash;
+use Illuminate\Support\Facades\Redirect;
+
+class SprintStoreActionHandler {
+	private $userPhabricatorAPI;
+	private $botPhabricatorAPI;
+	private $redirect;
+	private $sprint;
+	private $user;
```

Comment:
I know that in this application no issue will occur at present, since this service is always invoked just once and then discarded. Still, I think it's good practice to avoid this kind of state in services wherever possible.